docs/build-on-polkadot/xcm/hrmp-channels/overview.md

```diff
@@ -0,0 +1,211 @@
+---
+title: HRMP Channels
+description: HRMP channels enable cross-chain communication in Polkadot, a temporary solution before the more efficient XCMP protocol is implemented.
+---
+
+# HRMP Channels
+
+## Introduction
+
+Polkadot is designed to enable seamless interoperability between its connected parachains. At the core of this interoperability is the Cross-Consensus Message Format (XCM), a standard language that allows parachains to communicate and interact with each other.
+
+The actual network-layer protocol responsible for delivering XCM-formatted messages between parachains is called the Cross-Chain Message Passing (XCMP) protocol. XCMP maintains messaging queues on the relay chain, serving as a bridge to facilitate cross-chain interactions.
+
+However, as XCMP is still under development, Polkadot has implemented a temporary substitute known as Horizontal Relay-routed Message Passing (HRMP). HRMP provides the same interface and functionality as the planned XCMP, but with a key difference - it stores all messages directly in the relay chain's storage, which is more resource-intensive.
+
+Once XCMP is fully implemented, HRMP will be deprecated in favor of the native XCMP protocol. XCMP will offer a more efficient and scalable solution for cross-chain message passing, as it will not require the relay chain to store all the messages.
+
+## Prerequisites (WIP)
+
+<!-- TODO -->
+
+## Establishing HRMP Channels
+
+To facilitate communication between parachains using the HRMP protocol, the parachains must explicitly establish communication channels by registering them on the relay chain.
+
+Downward and upward channels from and to the relay chain are implicitly available, meaning they do not need to be explicitly opened.
+
+Opening an HRMP channel requires the parachains involved to make a deposit on the relay chain. This deposit covers the expenses of using the relay chain's storage for the message queues associated with the channel.
```

Comment:
Is this deposit a fixed amount, or may it vary depending on the network, the messages to be sent, and so on?